packages/turbo-repository/rust/src/lib.rs

```diff
@@ -73,13 +84,36 @@ impl Package {
         workspace_path: &AbsoluteSystemPath,
     ) -> Vec<Package> {
         let node = PackageNode::Workspace(PackageName::Other(self.name.clone()));
-        let ancestors = match graph.immediate_ancestors(&node) {
-            Some(ancestors) => ancestors,
+        let pkgs = match graph.immediate_ancestors(&node) {
```

Comment:
Are you sure you want just `immediate_ancestors`? What about transitive dependents? If you do in fact just want immediates, maybe a comment or update the method name to indicate that.

Reply:
just immediate for now. We are constructing the graph client side in our use case, we'll keep doing that for now and iterate here. I can't remember why I didn't do full tree on `ancestors` when I first did it, probably Rust-related hurdles.